### DISCUSSION TITLE
Curso html e css senac

### DISCUSSION THREAD

**Discussion by lynx0987** [frontendbr/forum#2316] [Discussão]
Olá, gostaria de saber se alguém fez o curso livre de html e css EAD do Senac e o que vcs acharam? E se possível quais cursos vocês recomendam?

https://www.ead.senac.br/cursos-livres/html-e-css-criacao-de-websites/

**Comment by felquis** [comment]
Não fiz, mas posso dizer que o primeiro curso que eu fiz foi bem ruim também e graças a Deus terminei teve até certificado.

**Comment by rackmorais** [comment]
Recomendo o curso da Oracle + alura, a didática desse curso é surreal, nunca aprendi tanto, mais vc tem q se programar pra estudar direitinho, pq ele dura só 6 meses e dps vc n tem acesso a conta, entao lembra de salvar os certificados. https://www.oracle.com/br/education/oracle-next-education/

Espero ter ajudado, boa sorte.
